homeworks/vadym.myroshnyk_VadimMir/homework_5/homework_5.js

```diff
@@ -0,0 +1,81 @@
+// tesk 1
+const someArr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+function sumArrayFunction() {
+    let sum = 0;
+    return someArr.map((item) => {
+        sum = item + sum;
+        return sum;
+    });
+}
+
+console.log(sumArrayFunction());
+
+// tesk 2
+const classes = ['header', 'menu', 'menu_item', 'tabs',
+    'tab_item', 'menu', 'link', 'tabs', 'tab_item', 'menu',
+    'menu_item', 'menu', 'menu_item'];
+
+function returnObjKey() {
```

Comment:
take as a common practice to write function which accept arguments, in this case the proper way would be to pass `classes` as argument